operator/controllers/utils.go

```diff
@@ -0,0 +1,127 @@
+package controllers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	flowv1beta1 "github.com/banzaicloud/logging-operator/pkg/sdk/api/v1beta1"
+	"github.com/banzaicloud/logging-operator/pkg/sdk/model/output"
+	loggingplumberv1alpha1 "github.com/mrsupiri/rancher-logging-explorer/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"net/http"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"time"
+)
+
+func banzaiTemplates(flow flowv1beta1.Flow, flowTest loggingplumberv1alpha1.FlowTest, extraLabels map[string]string) (flowv1beta1.Flow, flowv1beta1.Output) {
+	flowTemplate := flowv1beta1.Flow{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "logging.banzaicloud.io/v1beta1",
+			Kind:       "Flow",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s-test", flow.ObjectMeta.Name),
+			Namespace: flow.Namespace,
+			Labels: map[string]string{
+				"app.kubernetes.io/name":                flow.ObjectMeta.Name,
+				"app.kubernetes.io/managed-by":          "rancher-logging-explorer",
+				"app.kubernetes.io/created-by":          "logging-plumber",
+				"loggingplumber.isala.me/flowtest-uuid": string(flowTest.ObjectMeta.UID),
+				"loggingplumber.isala.me/flowtest":      flowTest.ObjectMeta.Name,
+			},
+		},
+		Spec: flowv1beta1.FlowSpec{
+			LocalOutputRefs: nil,
+			Match: []flowv1beta1.Match{{
+				Select: &flowv1beta1.Select{Labels: extraLabels},
+			}},
+		},
+	}
+
+	outTemplate := flowv1beta1.Output{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "logging.banzaicloud.io/v1beta1",
+			Kind:       "Output",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s-test", flow.ObjectMeta.Name),
+			Namespace: flow.Namespace,
+			Labels: map[string]string{
```

Comment:
this should come from a function

Reply:
This will get addressed in https://github.com/MrSupiri/rancher-logging-explorer/issues/16